mantis adapter test fails

## Type of issue
Bug

## Description
I've cloned prebid repository, installed node version v5.1.1, installed all dependencies and ran "gulp run-tests" command. And one test keeps failing:

Chrome 55.0.2883 (Linux 0.0.0) mantis adapter tests callBids should load script with relevant bid data FAILED
	AssertionError: expected 'https://mantodea.mantisadnetwork.com/website/prebid?referrer=http%3A%2F%2Flocalhost%3A9876%2F%3Fid%3D3053950&tz=-120&buster=1487265444745&secure=false&iframe=true&jsonp=mantis_jsonp%5B1%5D&property=1234&bids[0][bidId]=bidId1&bids[0][sizes][0][width]=728&bids[0][sizes][0][height]=90&bids[1][bidId]=bidId2&bids[1][sizes][0][width]=300&bids[1][sizes][0][height]=600&bids[1][sizes][1][width]=300&bids[1][sizes][1][height]=250&version=1' to match /tz=[0-9]+&/
	    at context.<anonymous> ( ... /test/spec/adapters/mantis_spec.js:142:27)


## Steps to reproduce
Clone a repo, install dependencies, run tests

### Expected results
All test pass successfully

### Actual results
One test fails

## Platform details
0.19.0
Chrome 55 / Fedora
node 5.1.1
npm 3.3.12

--- a/test/spec/adapters/mantis_spec.js
+++ b/test/spec/adapters/mantis_spec.js
@@ -97,7 +97,7 @@ describe('mantis adapter tests', function () {
 			var serverCall = adloader.loadScript.firstCall.args[0];
 
 			expect(serverCall).to.match(/buster=[0-9]+&/);
-			expect(serverCall).to.match(/tz=[0-9]+&/);
+			expect(serverCall).to.match(/tz=-?[0-9]+&/);
 			expect(serverCall).to.match(/secure=(true|false)&/);
 			expect(serverCall).to.string('property=1234&');
 			expect(serverCall).to.string('bids[0][bidId]=bidId1&');

